docs/GettingStartedDocs/Contributors/OPTEEGettingStarted.md

```diff
@@ -0,0 +1,69 @@
+# Getting Started with Open Enclave for OP-TEE OS
+
+The Open Enclave SDK as well as hosts and enclaves build via cross-compilation.
+Therefore, an ARM system, physical or emulated, is necessary only for execution,
+but not for development and building.
+
+## Platform Requirements
+
+- Ubuntu 16.04 LTS (64-bit) or 18.04 LTS (64-bit)
+
+## Clone the Open Enclave SDK
+
+```bash
+git clone --recursive https://github.com/openenclave/openenclave.git
```

Comment:
I think we should update instructions everywhere to use --recursive. I'll create an issue for this, as there is no need to do this in the scope of your PR

Reply:
Good idea! Thanks.